cblox/include/cblox/core/submap_collection.h

```diff
@@ -161,7 +184,9 @@ class SubmapCollection : public SubmapCollectionInterface {
   SubmapID active_submap_id_;
 
   // Submap storage and access
-  std::map<SubmapID, typename SubmapType::Ptr> id_to_submap_;
+  // TODO(alexmillane): Remove commented out version once aligned version works.
+  //std::map<SubmapID, typename SubmapType::Ptr> id_to_submap_;
```

Comment:
rm?

Reply:
@ntonci I was hunting down a bug, which I think I later found elsewhere when I wrote this... 

To my eye, this looks unnecessary now, because the map holds _pointers_ to submaps, we shouldn't need to align it right? Do you have an opinion on this?